File: tags/IVEF_0_1_0/ivef-objc/Classes/IVEFSchema.m

#import "IVEFSchema.h"


@implementation IVEFSchema

- (id) init {
    self = [super init];
    if (self != nil) {
        m_dateFormatter = [[NSDateFormatter alloc] init];
        [m_dateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS"];
    }
    return self;
}

-(NSString *) getElementFormDefault {

    return @"qualified";
}

-(NSString *) getTargetNamespace {

    return @"urn:http://www.ivef.org/XMLSchema/IVEF/1.1";
}

-(void) setAttributes:(NSDictionary *)attributeDict {

}

-(NSString *) XML {

    NSMutableString *xml = [NSMutableString stringWithString:@"<Schema"];
    [xml appendString:@">\n"];
    [xml appendString: @"</Schema>\n"];
    return xml;
}

-(NSString *) stringValueWithLead: (NSString *) lead {

    NSMutableString *str = [[[NSMutableString alloc] init] autorelease];
    [str setString: [lead stringByAppendingString:@"Schema\n"]];
    return str;
}


@end

File: OLD_tags/IVEF_0_1_0_RC3/ivef-qt/IVEFSchema.cpp

#include "IVEFSchema.h"

Schema::Schema() {

}

Schema::Schema(const Schema &val) : QObject() {

}

Schema & Schema::operator=(const Schema &val) {

    return *this;
}

QString Schema::getElementFormDefault() const {

    return "qualified";
}

QString Schema::getTargetNamespace() const {

    return "urn:http://www.ivef.org/XMLSchema/IVEF/1.1";
}

QString Schema::toXML() {

    QString xml = "<Schema";
    xml.append(">\n");
    xml.append( "</Schema>\n");
    return xml;
}

QString Schema::toString(QString lead) {

    QString str = lead + "Schema\n";
    return str;
}

